### PR TITLE
KRACOEUS-8873: Delivery info changes

### PR DESCRIPTION
**Pull request by gathreya**

### CODE REVIEW

File: coeus-impl/src/main/resources/ApplicationResources.properties [hunk 1]
```diff
@@ -809,6 +809,9 @@ error.proposalDevelopment.permission.submitToSponsor=User {0} does not have a pe
 message.institutionalproposal.not.created.inroute=The proposal has not yet completed the approval cycle, however it can be submitted to the sponsor as long as the original institutional proposal is versioned or if a new institutional proposal record is created.
 error.proposalDevelopment.resubmission.prompt.option.required=Please enter an institutional proposal number or select one of the other two options.
 
+#Delivery info
+message.proposalDevelopment.delivery.info.not.needed=<b>Please Note -</b><br> This proposal is being submitted via an S2S opportunity. As such, it will be submitted electronically. The delivery information on this page will not be needed
```
Comment (leoherbie) [inline coeus-impl/src/main/resources/ApplicationResources.properties]: Please close the br html tag.

Comment (leoherbie) [inline coeus-impl/src/main/resources/ApplicationResources.properties]: Missing period at the end of the last sentence

**Comment by gathreya** [comment]
added period.

File: coeus-impl/src/main/resources/ApplicationResources.properties [hunk 2]
```diff
@@ -809,6 +809,9 @@ error.proposalDevelopment.permission.submitToSponsor=User {0} does not have a pe
 message.institutionalproposal.not.created.inroute=The proposal has not yet completed the approval cycle, however it can be submitted to the sponsor as long as the original institutional proposal is versioned or if a new institutional proposal record is created.
 error.proposalDevelopment.resubmission.prompt.option.required=Please enter an institutional proposal number or select one of the other two options.
 
+#Delivery info
+message.proposalDevelopment.delivery.info.not.needed=<b>Please Note -</b><br> This proposal is being submitted via an S2S opportunity. As such, it will be submitted electronically. The delivery information on this page will not be needed.
```
Comment (leoherbie) [inline coeus-impl/src/main/resources/ApplicationResources.properties]: self close the br html tag

**Comment by gathreya** [comment]
just used p instead.

**Comment by gathreya** [comment]
Are there any other changes I need to make here or am I good?

**Pull request merged**